tests/unit/test_TOC.py

```diff
@@ -29,18 +29,24 @@
 
 ELEMS3 = (('PIL.Image.py', '/usr/lib/python2.7/encodings/__init__.py', 'PYMODULE'),)
 
+# Ignore deprecation warnings for the TOC class
+ignore_deprecation = pytest.mark.filterwarnings("ignore:TOC class is deprecated.")
```

Comment:
You can apply a pytest marker to a whole script by assigning it to `pytestmark`. Then you don't need the decorator on every test.
```suggestion
pytestmark = pytest.mark.filterwarnings("ignore:TOC class is deprecated.")
```

Reply:
Thanks, didn't know that!